### DISCUSSION TITLE
Stereo on OIRT

### DISCUSSION THREAD

**Discussion by andimik** [PE5PVB/TEF6686_ESP32#615] [Ideas]
During yesterday's Sporadic E I have received OIRT on various frequencies where TEF showed the stereo flag. I even didn't know that it was possible to use it in OIRT band. Then I found this useful page:

https://forums.stereotool.com/viewtopic.php?t=6233

But can you tell me, if TEF is using 38 kHz or 31.25 on OIRT?

**Comment by PE5PVB** [comment]
No, TEF doesn't support Polar stereo. But there are a lot of OIRT stations out there who use normal CCIR stereo and even RDS.

**Comment by andimik** [comment]
ok thanks
